call_apply_bind_exercise/callApplyBind.js

```diff
@@ -0,0 +1,68 @@
+var obj = {
+    fullName: "Harry Potter",
+    person: {
+        sayHi: function(){
+            return "This person's name is " + this.fullName
+        }.bind(obj)
```

Comment:
This doesn't work because obj won't be defined yet.  Here is a work around:

```
var obj = {
  fullName: "Harry Potter",
  person: {
    sayHi() {
      return `This person's name is ${this.fullName}`;
    }
  }
}
obj.person.sayHi = obj.person.sayHi.bind(obj);
```